app.js

```diff
@@ -1,13 +1,22 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controller");
-const { handleServerErrors } = require("./errors");
+const {
+  handleServerErrors,
+  handleCustomErrors,
+  handlePsqlErrors,
+} = require("./errors");
+const { getArticleByID } = require("./controllers/articles.controllers");
 
 const app = express();
 
 app.use(express.json());
```

Comment:
This shouldn't be here. How have we missed this until now?! 😁 Please only add code that is being used by the app. You don't need this until dealing with the client request body.